pipelines/tekton/gitops-update-pipeline/gitops-update-pipeline.yaml

```diff
@@ -27,6 +27,12 @@ spec:
     - name: gitTokenSecretKey
       default: token
       type: string
+    - name: yq-script
```

Comment:
Can we add a `description` field to this parameter just so anyone can understand the intent?

Reply:
Yes, great idea! Sorry, I added this a while back, but forgot to reply to your comment and mark it as resolved.